tests/e2e/steps_democracy.go

```diff
@@ -108,16 +115,13 @@ func stepsDemocracy(consumerName string) []Step {
 					},
 					// Check that the prop passed
 					Proposals: &map[uint]Proposal{
-						1: ParamsProposal{
-							Deposit:  10000001,
-							Status:   "PROPOSAL_STATUS_PASSED",
-							Subspace: "transfer",
-							Key:      "SendEnabled",
-							Value:    "true",
+						1: TextProposal{
```

Comment:
this proposal was enabling sends, seems important to keep

Reply:
Added neew prop type